src/components/pages/Dashboard/InstructorStats.js

```diff
@@ -22,16 +22,14 @@ const initialValues = {
   totalCourse: 0,
 };
 function InstructorStats(props) {
-  const { authState, authService } = useOktaAuth();
-  const { idToken } = authState;
+  // const { idToken } = authState;
   const dispatch = useDispatch();
   const [stats, setStats] = useState(initialValues);
 
   useEffect(() => {
     if (!props.user.name) {
       dispatch(getCurrentUser(idToken, authState, authService));
```

Comment:
We're getting errors here because `idToken`, `authState`, and `authService` are not defined.